transport/http/transport.go

```diff
@@ -80,6 +94,14 @@ func Tracer(tracer opentracing.Tracer) TransportOption {
 	}
 }
 
+// Socks5Proxy sets up outbounds connections to run through the specified proxy
+// of the format host:port
+func Socks5Proxy(hostPort string) TransportOption {
```

Comment:
Because the Socks5Proxy takes that format, and this value is most likely coming from somebody's yaml config, which is probably of the form:
```
proxy:
  socks5: localhost:<port>
```